src/init.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  BrowserClient,
+  makeFetchTransport,
+  defaultStackParser,
+  defaultIntegrations,
+} from '@sentry/browser';
+
+let sentryTracker: BrowserClient | null = null;
+
+export const init = (): void => {
```

Comment:
is the `init` name a convention? I'd change it to something like `sentryInit`

Reply:
hmm, I agree but we could append more to this ``init`` function.